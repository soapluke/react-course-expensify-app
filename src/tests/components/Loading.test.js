import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Loading from '../../components/Loading';

test('should render Loading correctly', () => {
    const wrapper = shallow(<Loading />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});