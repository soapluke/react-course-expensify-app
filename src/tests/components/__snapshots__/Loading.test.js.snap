// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loading correctly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/imgs/loader.gif"
  />
</div>
`;
